pom.xml

```diff
@@ -62,6 +62,7 @@
     <dependency>
       <groupId>org.hibernate</groupId>
       <artifactId>hibernate-validator</artifactId>
+      <version>6.0.17.Final</version>
```

Comment:
Pls add the pointer to stackoverflow here

Reply:
https://stackoverflow.com/questions/46482088/spring-boot-starter-web-hibernate-validator-dependency-missing-on-mac